docs/buildah-config.md

```diff
@@ -77,8 +78,9 @@ executed together.
 
 **--env** *var=value*
 
-Add a value (e.g. name=*value*) to the environment for containers based on any
+Add a value (e.g. var=*value*) to the environment for containers based on any
 images which will be built using the specified container. Can be used multiple times.
+If *env* has a trailing `-`, then the *env* is removed from the config.
```

Comment:
but there's no `*env*` prior to this.  I think we might want to change var=value to env=value or change `*env*` here to var.  Please add an example.